css/media.css

```diff
@@ -0,0 +1,1039 @@
+@media screen and (max-width: 1100px) {
+  .container {
+    max-width: 970px;
+  }
+  
+  .video-block {
+    padding: 55px 30px;
+    min-height: 441px;
+  }
+
+  .video-block__pic {
+    display: inline-block;
+    width: 579px;
+  }
+}
+
+@media screen and (max-width: 1024px) {  
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected newline after "{" of a multi-line block (block-opening-brace-newline-after)](https://app.codacy.com/gh/WebHeroSchool/Axion2/pullRequest?prid=5654145)